demosplan/DemosPlanCoreBundle/ResourceTypes/CustomerContactResourceType.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\ResourceTypes;
+
+use DemosEurope\DemosplanAddon\Contracts\ResourceType\CreatableDqlResourceTypeInterface;
+use DemosEurope\DemosplanAddon\Contracts\ResourceType\UpdatableDqlResourceTypeInterface;
+use DemosEurope\DemosplanAddon\Logic\ResourceChange;
+use demosplan\DemosPlanCoreBundle\Entity\User\SupportContact;
+use demosplan\DemosPlanCoreBundle\Logic\ApiRequest\PropertiesUpdater;
+use demosplan\DemosPlanCoreBundle\Logic\ApiRequest\ResourceType\DeletableDqlResourceTypeInterface;
+use demosplan\DemosPlanCoreBundle\Logic\ApiRequest\ResourceType\DplanResourceType;
+use demosplan\DemosPlanCoreBundle\Logic\EmailAddressService;
+use EDT\PathBuilding\End;
+use Webmozart\Assert\Assert;
+
+/**
+ * @template-implements UpdatableDqlResourceTypeInterface<SupportContact>
+ * @template-implements CreatableDqlResourceTypeInterface<SupportContact>
+ * @template-implements DeletableDqlResourceTypeInterface<SupportContact>
+ *
+ * @template-extends DplanResourceType<SupportContact>
+ *
+ * @property-read End                      $title
+ * @property-read End                      $phoneNumber
+ * @property-read End                      $text
+ * @property-read End                      $visible
+ * @property-read EmailAddressResourceType $eMailAddress
+ * @property-read CustomerResourceType     $customer
+ */
+class CustomerContactResourceType extends DplanResourceType implements CreatableDqlResourceTypeInterface, DeletableDqlResourceTypeInterface, UpdatableDqlResourceTypeInterface
```

Comment:
Due to local environment issues I was unable to finish the tests for this resource type. But however, tests are missing.